views/js/controller/creator/index.js

```diff
@@ -95,14 +95,14 @@ define([
             //load plugins dynamically
             if (config) {
                 if(config.plugins){
-                    _.forEach(config.plugins, function (plugin) {
+                    _.forEach(config.plugins, plugin => {
```

Comment:
lodash could be already removed

Reply:
`plugins` may be an `Object` not an `Array`